locales/fr/translation.json

```diff
@@ -57,59 +57,59 @@
-  "Basal to bolus insulin ratio": "",
-  "Basal:Bolus Ratio": "",
-  "Basals": "",
+  "Basal": "Basale",
+  "Basal : Bolus": "Basal : Bolus",
+  "Basal Events": "Evénements basal",
+  "Basal insulin keeps your glucose within your target range when you are not eating.  Bolus insulin is mostly used to cover the carbohydrates you eat or to bring a high glucose back into your target range. This ratio allows you to compare how much of the insulin you are taking is used for each purpose. We will only show numbers if there is enough basal data - 24 hours in the one day view and 7 days in the two week view.": "L'insuline basale maintient votre glycémie en normoglycémie entre vos repas. Les bolus sont communément utilisés pour couvrir l'apport de glucides lors des repas ou en hyperglycemie pour revenir à votre cible. Ce ratio vous permet de connaitre la répartition de l'insuline pour votre traitement. Nous afficherons les valeurs seulement si nous avons suffisament de données - 24 heures pour une journée et 7 jours pour la vue bihebdomadaire.",
+  "Basal Rates": "Débits de base",
+  "Basal to bolus insulin ratio": "Ratio d'insuline Basal / Bolus",
+  "Basal:Bolus Ratio": "Ratio Basal:Bolus",
+  "Basals": "Basales",
   "Basics": "Bases",
   "Because {{source}} represents extended boluses that cross midnight as two boluses, this bolus could be part of a dual-wave bolus, not an independent square-wave bolus as represented here.": "",
-  "Because of how {{source}} reports bolus and wizard data, we can't be 100% certain that the bolus wizard information here (e.g., carbs, suggested dose) corresponds with the bolus.": "",
-  "Because of how {{source}} reports bolus data, normal and square-wave boluses may not be properly combined to appear as a dual-wave bolus.": "",
-  "Because of how {{source}} reports the data, we could not determine the duration of this basal rate segment.": "",
-  "Because there was a time change during this basal, we split the basal into two segments.": "",
-  "below": "",
-  "between": "",
+  "Because of how {{source}} reports bolus and wizard data, we can't be 100% certain that the bolus wizard information here (e.g., carbs, suggested dose) corresponds with the bolus.": "En raison des {{source}} informations de bolus et de l'assistant bolus remontées, nous ne sommes pas certains à 100% que les informations de l'assistant bolus (e.g., glucides, dose proposée) correspondent à ce bolus.",
+  "Because of how {{source}} reports bolus data, normal and square-wave boluses may not be properly combined to appear as a dual-wave bolus.": "En raison des {{source}} informations des bolus remontées, il se peut que les bolus standard et les bolus carrés ne soient pas correctement affichés et apparaissent comme un bolus duo.",
+  "Because of how {{source}} reports the data, we could not determine the duration of this basal rate segment.": "En raison des {{source}} données remontées, nous ne pouvons déterminer la durée de ce débit de base. ",
```

Comment:
```suggestion
  "Because of how {{source}} reports the data, we could not determine the duration of this basal rate segment.": "En raison des données remontées par {{source}}, nous ne pouvons déterminer la durée de ce débit de base. ",
```